nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml

```diff
@@ -120,4 +124,99 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+
+
+    <profiles>
+        <profile>
+            <!-- Custom profile for Groovy tests -->
+            <id>groovyUnitTest</id>
+            <activation>
+                <!--<property>-->
+                <!--<name>env.USER</name>-->
+                <!--<value>alopresto</value>-->
+                <!--</property>-->
+                <!-- Change to read from environment variable -->
+                <property>
+                    <name>groovy</name>
+                    <value>test</value>
+                </property>
+            </activation>
+
+            <dependencies>
+                <dependency>
+                    <groupId>org.codehaus.groovy</groupId>
+                    <artifactId>groovy-all</artifactId>
+                    <version>2.4.5</version>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>1.8</version>
+                        <executions>
+                            <execution>
+                                <id>add-source</id>
+                                <phase>generate-sources</phase>
+                                <goals>
+                                    <goal>add-source</goal>
+                                </goals>
+                                <configuration>
+                                    <sources>
+                                        <source>src/main/groovy</source>
+                                    </sources>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>add-test-source</id>
+                                <phase>generate-test-sources</phase>
+                                <goals>
+                                    <goal>add-test-source</goal>
+                                </goals>
+                                <configuration>
+                                    <sources>
+                                        <source>src/test/groovy</source>
+                                    </sources>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <version>3.1</version>
+                        <configuration>
+                            <compilerId>groovy-eclipse-compiler</compilerId>
+                            <source>1.8</source>
+                            <target>1.8</target>
```

Comment:
Is this setting the Java source/target to 1.8? Or is this for the groovy compiler only? Need to ensure that we are only requiring Java 1.7.